tmt/utils.py

```diff
@@ -1478,15 +1633,12 @@ def _load(self, text):
         section = re.compile(r"\n\[([^\]]+)\][ \t]*\n", re.MULTILINE)
         parts = section.split(matched.groups()[1])
         # Detect the version
-        try:
-            self.version(int(re.search(
-                r"version (\d+)", parts[0]).groups()[0]))
-            log.debug(
-                "Detected StructuredField version {0}".format(self.version()))
```

Comment:
The two lines above seem to be lost.

Reply:
Just one of them, `log.debug` call. `self.version()` call is on line 1652.

Fixed in 3a48196f2d8f4c196850c807abcc49f325198e6f.